_posts/2021-01-05-Jamulus-Sound-Devices.md

```diff
@@ -218,12 +218,14 @@ Sounds great, can achieve 32 frame buffer and works on *Windows* and *Linux*.
 
 **[ZOOM LiveTrak L-8](https://zoomcorp.com/en/us/digital-mixer-multi-track-recorders/digital-mixer-recorder/LIVETRAK-L-8/)**, Mixer
 
-**Windows**: Latency is around 19ms (measured with local server with ping time of 0 ms) using "L-8 Driver" (ASIO, 07/02/2020) from [zoomcorp.com](https://zoomcorp.com/)
+**Windows**: Latency is around 19ms (measured with local server with ping time of 0 ms) using "L-8 Driver" (ASIO, 07/02/2020) from [zoomcorp.com](https://zoomcorp.com/). Driver can be downloaded from link to device and clicking "Support & Downloads".
 
 **macOS**: Not yet tested
 
 **Linux**: Not yet tested
 
+***
+
```

Comment:
Thank you. This looks good. I‘m a bit unsure: why did you add a line (`***`) here

Reply:
Just to normalize the formatting with a horizontal line between each device. Ignore it, if you think it was not a good idea.

Reply:
I'm open for suggestions. You could also add it on other places if you wish